Include/Acidanthera/Library/OcConfigurationLib.h

```diff
@@ -474,11 +474,13 @@ OC_DECLARE (OC_NVRAM_LEGACY_ENTRY)
   OC_MAP (OC_STRING, OC_NVRAM_LEGACY_ENTRY, _, __)
 OC_DECLARE (OC_NVRAM_LEGACY_MAP)
 
+//
+// Modifications to OC_NVRAM_CONFIG require updating OC_VARIABLE_RUNTIME_PROTOCOL_REVISION.
```

Comment:
Do we pass this structure to the driver now?

Reply:
With the current commit, I now only pass OC_NVRAM_LEGACY_MAP (which as far as I can see is required), not all of OC_NVRAM_CONFIG.